### PR TITLE
CASSANDRA-18896 ClientRequestSize metrics should not treat CONTAINS restrictions as being equality-based

### PR DESCRIPTION
**Pull request by maedhroz**

### CODE REVIEW

**Comment by maedhroz** [comment]
https://app.circleci.com/pipelines/github/maedhroz/cassandra/776/workflows/8686837d-d30e-4854-8b7d-8608b2417323

**Review by blerer — APPROVED** [review]

**Comment by maedhroz** [comment]
Circle run looks clean

File: test/unit/org/apache/cassandra/metrics/ClientRequestRowAndColumnMetricsTest.java [hunk 1]
```diff
@@ -620,6 +620,6 @@ public void shouldRecordReadMetricsForContainsQuery() throws Throwable
 
         assertEquals(1, ClientRequestSizeMetrics.totalRowsRead.getCount());
         // The filtering term is provided by the client in the request, so we don't consider that column read.
```
Comment (maedhroz) [inline test/unit/org/apache/cassandra/metrics/ClientRequestRowAndColumnMetricsTest.java:622]: Will fix this comment on commit...

**Comment by maedhroz** [comment]
Committed as https://github.com/apache/cassandra/commit/256e39fc62c0b0ce576fd29849672e6323eecea4

**Pull request closed without merging**